.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "standard-react"
+  ],
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "rules": {
+    "react/jsx-indent": [ 2, 4 ]
```

Comment:
Looks like it.  According to https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-indent.md "The default style is 4 spaces.".